client/cypress/integration/query/parameter_spec.js

```diff
@@ -1,6 +1,22 @@
 import { createQuery } from '../../support/redash-api';
 
 describe('Parameter', () => {
+  const expectDirtyStateChange = (edit) => {
```

Comment:
Cool idea :+1:. Only thing is that you're actually expecting "the dirty state", not it to change :sweat_smile:. Anyway, no big deal on this one.

Reply:
My intention was that it changes from false to true :) I'll leave it though.
I wish I could use `expect(paramEdit).to.change(getDirtyState);` 😭